### PR TITLE
Add target _blank to all external links

### PR DESCRIPTION
**Pull request by stormfar**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ date: 2017-01-29
...
 
 <p class="center-link">
-  <a href="https://www.nytimes.com/2017/01/28/us/us-immigration-ban.html?pagewanted=all">Read stories from banned travelers and asylum-seekers</a>
+  <a target="_blank" href="https://www.nytimes.com/2017/01/28/us/us-immigration-ban.html?pagewanted=all">Read stories from banned travelers and asylum-seekers</a>
 </p>
 <br/>
 While we can’t directly change the president’s decision, we CAN offer urgent help to those who need it.
 
-[\#NoMuslimBan](https://twitter.com/search?q=%23nomuslimban) [#NoBanNoWall](https://twitter.com/search?q=%23nobannowall) [#Resist](https://twitter.com/search?q=%23resist)
+<a href="https://twitter.com/search?q=%23resist" target="_blank">\#NoMuslimBan](https://twitter.com/search?q=%23nomuslimban) [#NoBanNoWall](https://twitter.com/search?q=%23nobannowall) [#Resist</a>
```
Comment (RichardLitt) [inline index.md]: This line is not ideal. The regex grabbed all of it. Do you know how to fix?

File: index.md [hunk 2]
```diff
@@ -129,13 +129,13 @@ Stranded By Trump was made by a group of Americans (mostly) traveling in Buenos
 
 We are nominally non-partisan, but we acknowledge that Trump's executive orders were racist, xenophobic, alienating, and decidedly un-American. We believe that no one person is illegal, and that America is best when it is welcoming all.
 
-Site idea and planning by [@richlitt](https://twitter.com/richlitt). The logo is by [Janne Wolterbeek](http://www.waldbach.nl/). Others (anonymous) helped out.
+Site idea and planning by <a href="http://www.waldbach.nl/" target="_blank">. Others (anonymous" target="_blank">@richlitt](https://twitter.com/richlitt). The logo is by [Janne Wolterbeek</a> helped out.
```
Comment (RichardLitt) [inline index.md]: Same here.

File: index.md [hunk 3]
```diff
@@ -38,88 +38,88 @@ If you are turned back:
...
 
 # For Those Who Want to Help
 
 Here’s what you can do to help those in desperate need right now:
 
-**1. Offer housing** on [Airbnb](https://www.airbnb.com/disaster) or other platforms
+**1. Offer housing** on <a href="https://www.airbnb.com/disaster" target="_blank">Airbnb</a> or other platforms
 
-**2. If you know a green card holder who is stuck outside the U.S.** with tickets to return to IAD -- please let [Simon Sandoval-Moshenberg](mailto:simon@justice4all.org) know ASAP. He is the Director of the [Immigrant Advocacy Program](https://www.justice4all.org/immigrant-advocacy-program/) at the Legal Aid Center in Washington, D.C.
+**2. If you know a green card holder who is stuck outside the U.S.** with tickets to return to IAD -- please let <a href="https://www.justice4all.org/immigrant-advocacy-program/" target="_blank">Simon Sandoval-Moshenberg](mailto:simon@justice4all.org) know ASAP. He is the Director of the [Immigrant Advocacy Program</a> at the Legal Aid Center in Washington, D.C.
```
Comment (RichardLitt) [inline index.md]: Same.

**Review by RichardLitt — COMMENTED** [review]
Needs some small fixes.

**Comment by stormfar** [comment]
Fixed.

**Comment by RichardLitt** [comment]
Merged in b463c077c56a8746338a9449125c8c11c1d3a532. Thank you!

**Pull request closed without merging**